modules/ROOT/examples/live-demos/comments-callback/example.js

```diff
@@ -169,15 +169,34 @@ const tinycomments_lookup = ({ conversationUid }, done, fail) => {
     });
 };
 
+const tinycomments_fetch = (conversationUids, done, fail) => {
```

Comment:
Needs to be updated to match `tinycomments_fetch` code in index.js

Reply:
Updated in [this comment](https://github.com/tinymce/tinymce-docs/pull/3458/commits/2e3f04df39bf985d1d95d1aece63985f3d7deeff) to match index.js